frontend/public/components/markdown-view.tsx

```diff
@@ -181,6 +183,8 @@ const IFrameMarkdownView: React.FC<InnerSyncMarkdownProps> = ({
       setFrameHeight(
         frame.contentWindow.document.body.firstElementChild.scrollHeight + (exactHeight ? 0 : 15),
       );
+      const htmlTagElement = frame?.contentDocument?.getElementsByTagName('html')[0];
+      updateThemeClass(htmlTagElement, theme, themeLoaded);
```

Comment:
if the frame or contentDocument, or.... could be optional we should check that htmlTagElement is defined before calling `updateThemeClass, or make this check in `updateThemeClass`, or?

Reply:
Added checks in `updateThemeClass`